packages/@aws-cdk/aws-backup/README.md

```diff
@@ -218,3 +236,27 @@ const role = new iam.Role(this, 'Access Role', { assumedBy: new iam.ServicePrinc
 
 importedVault.grant(role, 'backup:StartBackupJob');
 ```
+
+## Backup selection IAM role policies
+
+When creating a new `BackupSelection`, if no user IAM role is provided, a new
+IAM role with a managed policy for backup will be created for the selection. 
+
+All policies, by default, include the `AWSBackupServiceRolePolicyForBackup` AWS
```

Comment:
An extra reference never hurts, e.g. https://docs.aws.amazon.com/aws-backup/latest/devguide/iam-service-roles.html

Reply:
Okay - will add in - thanks